src/analysis/ReachingDefinitions/Ssa/SparseRDGraphBuilder.h

```diff
@@ -51,6 +51,8 @@ class SparseRDGraphBuilder
     {
         // restart state
```

Comment:
This would use some comment in the commit message (I do not understand this commit thoroughly).

Reply:
The rationale for the rename is, that the algorithm doesn't actually build SSA form in any way, so I found it misleading to name it `SSA Builder`. So I came up with the name `Sparse RD Graph` and from there `SRG builder`.
I should probably add that to the commit message to clarify my goals here.